languages/rust/exercises/concept/entry-api/.meta/example.rs

```diff
@@ -0,0 +1,23 @@
+use std::collections::HashMap;
+
+pub fn construct_note(magazine: &[&str], note: &[&str]) -> bool {
+    let mut counts = HashMap::with_capacity(magazine.len());
+
+    for m in magazine {
+        counts.entry(*m)
+            .and_modify(|v| *v += 1)
+            .or_insert(1);
```

Comment:
```suggestion
        *counts.entry(*m).or_default() += 1;
```

Shorter and more explicit.